org-gen/repos.yaml

```diff
@@ -0,0 +1,252 @@
+repos:
+  api:
+    allow_merge_commit: false
+    allow_rebase_merge: false
+    allow_squash_merge: false
+    description: API definitions for the Istio project
+    has_projects: false
+    has_wiki: false
+  bots:
+    allow_merge_commit: false
+    allow_rebase_merge: false
+    allow_squash_merge: false
+    description: Source code for bots used by the Istio project.
+    has_issues: false
+    has_projects: false
+    has_wiki: false
+  bottestrepo:
```

Comment:
we have this repo? :)

Reply:
We do. A few of us use it to create tests for prow jobs.

Reply:
Thanks